corehq/ex-submodules/casexml/apps/phone/data_providers/case/clean_owners.py

```diff
@@ -78,37 +71,37 @@ def get_payload(self):
             # todo: see if we can avoid wrapping - serialization depends on it heavily for now
             case_batch = filter(
                 partial(case_needs_to_sync, last_sync_log=self.restore_state.last_sync_log),
-                [CommCareCase.wrap(doc) for doc in get_docs(CommCareCase.get_db(), ids)
-                 if not is_deleted(doc)]
+                [case for case in CaseAccessors(self.restore_state.domain).get_cases(ids)
+                 if not case.is_deleted]
```

Comment:
Added [here](https://docs.google.com/document/d/1XX_WCEf8ozxt7cJoigu3R03W9vPqY45P7vFFY3stVO4/edit) so we don't forget about it.